tavla/app/components/PreviewCarousel/index.tsx

```diff
@@ -67,7 +67,7 @@ function PreviewCarousel({ boards }: { boards: TBoard[] }) {
     if (!currentBoard) return null
     return (
         <>
-            <div className="flex flex-row">
+            <div className="flex w-1/2 flex-row justify-between xl:w-1/2">
```

Comment:
Nå blir den vel alltid 1/2? 🤔 

Kunne vi kanskje satt breddel på div-en som går rundt hele innholdet, i stedet for karusellen og teksten hver for seg?